examples/webhook-signing/nextjs/api-router/route.ts

```diff
@@ -0,0 +1,43 @@
+import { NextRequest, NextResponse } from 'next/server';
+import Stripe from 'stripe';
+import { headers } from 'next/headers'
+import { getErrorMessage, logError } from '@/lib/error-methods';
```

Comment:
Can we remove this import as it looks like code specific to your application and may confuse others.